.eslintrc

```diff
@@ -24,14 +24,13 @@
       "unicodeCodePointEscapes": true,
       "globalReturn": false,
       "jsx": true,
-      "experimentalObjectRestSpread": true
+      "experimentalObjectRestSpread": true,
     }
   },
   "rules": {
     "indent": ["error", 2],
-    "react/jsx-indent-props":  ["error", 2],
-    "react/jsx-indent": ["error", 2],
-    "semi": [2]
+    // "react/jsx-indent-props":  ["error", 2],
+    // "react/jsx-indent": ["error", 2],
```

Comment:
Another issue: the linter won't check the JSX indentation until I sort this out.

Also looks like I clobbered the `semi` rule, I'll at least add that back.